planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/manager.cpp

```diff
@@ -352,12 +353,22 @@ void LaneChangeModuleManager::updateModuleParams(const std::vector<rclcpp::Param
     updateParam<int>(parameters, ns + "lon_acc_sampling_num", longitudinal_acc_sampling_num);
     if (longitudinal_acc_sampling_num > 0) {
       p->trajectory.lon_acc_sampling_num = longitudinal_acc_sampling_num;
+    } else {
+      RCLCPP_WARN(
+        rclcpp::get_logger("lane_change"),
+        "Parameter 'lon_acc_sampling_num' is not updated because the value (%d) is not positive",
+        longitudinal_acc_sampling_num);
     }
 
     int lateral_acc_sampling_num = 0;
     updateParam<int>(parameters, ns + "lat_acc_sampling_num", lateral_acc_sampling_num);
     if (lateral_acc_sampling_num > 0) {
       p->trajectory.lat_acc_sampling_num = lateral_acc_sampling_num;
+    } else {
+      RCLCPP_WARN(
+        rclcpp::get_logger("lane_change"),
```

Comment:
please use node->get_logger.

Reply:
node is not passed and it prevents from using node in this function :thinking:

Reply:
you can use member variable `node_`.